.github/workflows/native-image.yaml

```diff
@@ -0,0 +1,65 @@
+on:
+  # This allows manual activation of this action for testing.
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron: '0 2 * * 1,2,3,4,5'
+name: native-image
+env:
+  GOOGLE_CLOUD_PROJECT: "span-cloud-testing"
+  GOOGLE_CLOUD_INSTANCE: "pgadapter-testing"
+  GOOGLE_CLOUD_DATABASE: "testdb_integration"
+  GOOGLE_CLOUD_ENDPOINT: "spanner.googleapis.com"
```

Comment:
Is it fine to to put these in public repo?

Reply:
The endpoint is public information. It's for example also included in the generated client for each programming language.
The project, instance and database names are also included in many other repositories.